### DISCUSSION TITLE
Tags Page

### DISCUSSION THREAD

**Discussion by Xhanti** [thegeeklab/hugo-geekblog#167] [Q&A] [answered]
Hi, 

I wanted to know if there is a way to get a page that can list all the tags on a site?
I saw this page: https://hugo-geekblog.geekdocs.de/tags/
But it looks like this:

![image](https://user-images.githubusercontent.com/2096304/151428449-15192982-6c65-41ed-8b98-d649a77fe925.png)

Any insight you could provide would be much appreciated.
Thank you again for the excellent theme

**Comment by xoxys** [comment]
Hi,

All tags are added to the main menu already, e.g. https://hugo-geekblog.geekdocs.de/tags/development/ and will list all posts with the corresponding tag. What would you expect to be shown on the tags page? Just all tags again linking to the tags page?

**Reply by Xhanti** [reply]
Hi,

Yes a page that links to the tags page. I have this setting enabled  geekblogTagsToMenu = false. I want a static navigation menu and a separate page for the tags. I have too many tags for the bar to work for me.

Regards

**Reply by xoxys** [reply]
Sounds reasonable and should be easy to fix. Will look into it later.

**Reply by Xhanti** [reply]
Thank you :)

**Comment by xoxys** [comment, marked as answer]
Added by https://github.com/thegeeklab/hugo-geekblog/pull/172
